src/os.cpp

```diff
@@ -240,3 +240,22 @@ void OS::printData() const
     LOG_DEBUG("Minor Version: {}", m_minorVersion);
 }
 #endif
+
+#ifdef BUILD_TESTING
+#include <cloysterhpc/tests.h>
+
+TEST_SUITE("Test OS class")
```

Comment:
I didn't get this test also. What's testing exactly? Because those values are get during runtime and now during compile time.